src/compo/mopitt_co_nc2ioda.py

```diff
@@ -131,25 +137,28 @@ def _read(self):
             #now calculate the apriori term to pass to UFO and ensure single precision
             ap_tc=np.zeros(len(lats))
             for lev in range(nlevs):
-               ap_tc = ap_tc + ak_tc_dimless[:,lev]*(pr_gd[:,lev]-pr_gd[:,lev+1])*xa_gd[:,lev]
+               ap_tc = ap_tc + vmr2col*ak_tc_dimless[:,lev]*(pr_gd[:,lev]-pr_gd[:,lev+1])*xa_gd[:,lev]
             ap_tc = xa_tc - ap_tc
             ap_tc = ap_tc.astype('float32')
             flg = qa == 0
 
+            #unit conversion from molecules/cm2 to mol/m2
+            u_conv = avogadro / scm2sm
+
             if first:
                 # add metadata variables
                 self.outdata[('datetime', 'MetaData')] = times[flg]
                 self.outdata[('latitude', 'MetaData')] = lats[flg]
                 self.outdata[('longitude', 'MetaData')] = lons[flg]
-                self.outdata[('apriori_term', 'MetaData')] = ap_tc[flg]
+                self.outdata[('apriori_term', 'MetaData')] = ap_tc[flg]/u_conv
                 for k in range(nlevs):
                     varname_ak = ('averaging_kernel_level_'+str(k+1), 'MetaData')
                     self.outdata[varname_ak] = ak_tc_dimless[:,k][flg]
                     varname_pr = ('pressure_level_'+str(k+1), 'MetaData')
-                    self.outdata[varname_pr] = pr_gd[:,k][flg]
+                    self.outdata[varname_pr] = hPa2Pa*pr_gd[:,k][flg]
```

Comment:
I bet this doesn't pass coding norms. Might want to check.

Reply:
good point. do you know how to know which lines?

Reply:
See my comment in the discussion. Look here:
https://cdash.jcsda.org/testDetails.php?test=4907051&build=63434
There are ~50 'complaints' by the coding norms test. Also the new test itself fails because of the a priori variable being ~20 orders of magnitude different.

Reply:
Yes this was wrong before so I changed it , but normallly the testoutput has been updated. Let me double check that might be it.

Reply:
test is passing fine on Orion, local and remote must not in sync but git say it's ok...

Reply:
Let me try tomorrow to build/test and see what I get just to be sure

Reply:
see commit f2addabcec634975cda5464785533d22489c7a49